### PR TITLE
Clean up landing page + instructions

### PR DESCRIPTION
**Pull request by scottyhq**
More work on #55

- [x] add announcement banner (link to user survey)
- [x] move Tutorial Setup from 'Fundamentals/Introduction' to front page
- [x] clean up overview notebook
- [x] use jupyterbook cell metadata tags for scrolling outputs, etc. on website

### CODE REVIEW

**Comment by scottyhq** [comment]
@dcherian I simplified the `Overview` notebook a bit, mainly reducing the length of `# Titles` so that navigating the notebook with the right hand browser is easier. Also removed the creation of a dask cluster as I think that can get kind of confusing (processes vs threads), so in the interest of an overview I thought the simplest default configuration (just specifying `chunks`) would be good. Could add a link to https://docs.dask.org/en/stable/scheduling.html and mention the single-machine scheduler is used and go into more detail into intermediate or advanced sections

File: _config.yml [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+title: ''
+author: The Xarray Community
 copyright: '2022'
 logo: images/logo.png
 only_build_toc_files: true
 
+# See https://jupyterbook.org/customize/config.html#add-a-link-to-your-repository
+html:
+  # NOTE: this announcement shows up on all pages
+  announcement: 'ℹ️ The 2022 Xarray User Survey is out! Please take ~5 minutes to help improve Xarray! <a href="https://docs.google.com/forms/d/e/1FAIpQLSfnMd8UsC1XP1lPuFczl148VfpmwnFu4a0Z94odt1L6U0R0Pw/viewform">LINK</a>'
```
Comment (dcherian) [inline _config.yml:11]: Nice. thanks!

**Review by dcherian — APPROVED** [review]

File: fundamentals/00_overview.ipynb [hunk 2]
```diff
@@ -1,196 +0,0 @@
...
-   "outputs": [],
-   "source": [
-    "print(\"Hello, world!\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Going Deeper\n",
```
Comment (dcherian) [inline fundamentals/00_overview.ipynb:144]: I think this is valuable and could go as a separate heading under "Overview".

Comment (scottyhq) [reply fundamentals/00_overview.ipynb:144]: good catch i forgot to copy that over. i'll update the links to the repositories b/c pangeo binder is still broken

**Comment by scottyhq** [comment]
Going to merge this in now that CI is passing!

File: workshops/scipy2022/xarray_ecosystem.ipynb [hunk 3]
```diff
@@ -63,22 +63,6 @@
     "\n",
```
Comment (keewis) [inline workshops/scipy2022/xarray_ecosystem.ipynb:63]: note that we lifted that restriction, the import order is not important anymore

**Pull request merged**